### ISSUE TITLE
Automapper on 64 bit device with Xamarin Unified

### ISSUE THREAD

**Issue by jakkaj** [AutoMapper/AutoMapper#669]
When running AutoMapper on a 64 bit device the app crashes. 

Running the same app in the simulator or on a 32 bit device does not have this problem. 

It's interesting because the problem manifests somewhere else - in this case the file system write operation... took quite a while to track it back to AutoMapper.

```
2015-01-24 12:35:23.660 AutomapperProblemReplicate[601:149248] critical: Stacktrace:
2015-01-24 12:35:23.661 AutomapperProblemReplicate[601:149248] critical:   at <unknown> <0xffffffff>
2015-01-24 12:35:23.661 AutomapperProblemReplicate[601:149248] critical:   at (wrapper managed-to-native) object.__icall_wrapper_mono_array_new_specific (intptr,int) <0xffffffff>
2015-01-24 12:35:23.662 AutomapperProblemReplicate[601:149248] critical:   at System.IO.StreamWriter.Initialize (System.Text.Encoding,int) [0x00017] in /Developer/MonoTouch/Source/mono/mcs/class/corlib/System.IO/StreamWriter.cs:77
2015-01-24 12:35:23.662 AutomapperProblemReplicate[601:149248] critical:   at System.IO.StreamWriter..ctor (string,bool,System.Text.Encoding,int) [0x00076] in /Developer/MonoTouch/Source/mono/mcs/class/corlib/System.IO/StreamWriter.cs:146

```

Full stack is at the bottom. 

Replication is easy - create a new iOS test project, and add the following test. 

```
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using AutoMapper;
using Foundation;
using NUnit.Framework;
using UIKit;

namespace AutomapperProblemReplicate.Tests
{
    [TestFixture]
    public class ReplicateTest
    {
        [Test]
        public void CauseCrash()
        {
            var text = "this is text";
            var path = _getPath("SomeFile");
            _createDirForFile(path);
            for (var i = 0; i < 500; i++)
            {
                text += i;
                
                Mapper.CreateMap<TestClass, TestClass>();
                File.WriteAllText(path, text);
            }
        }

        private static string path = null;

        private string _getPath(string fileName)
        {
            if (path == null)
            {
                var documents = NSFileManager.DefaultManager.GetUrls(NSSearchPathDirectory.CachesDirectory, NSSearchPathDomain.User)[0];
                path = documents.Path;
            }

            var result = Path.Combine(path, fileName);
            return result;
        }

        void _createDirForFile(string fileName)
        {
            var dir = Path.GetDirectoryName(fileName);

            //Debug.WriteLine("Creating Directory: {0}", fileName);

            if (dir == null)
            {
                return;
            }

            if (!Directory.Exists(dir))
            {
                Directory.CreateDirectory(dir);
            }
        }
    }

    public class TestClass
    {
        public string Name { get; set; }
    }
}
```

Full stack dump

```
2015-01-24 12:35:23.328 AutomapperProblemReplicate[601:149248] [Runner executing:	AutomapperProblemReplicate.Tests.ReplicateTest.CauseCrash]
2015-01-24 12:35:23.328 AutomapperProblemReplicate[601:149248] [MonoTouch Version:	8.6.1]
2015-01-24 12:35:23.329 AutomapperProblemReplicate[601:149248] [Assembly:	Xamarin.iOS.dll (64 bits)]
2015-01-24 12:35:23.330 AutomapperProblemReplicate[601:149248] [GC:	sgen+NewRefCount]
2015-01-24 12:35:23.331 AutomapperProblemReplicate[601:149248] [iPhone:	iPhone OS v8.1.2]
2015-01-24 12:35:23.334 AutomapperProblemReplicate[601:149248] [Device Name:	iJak]
2015-01-24 12:35:23.335 AutomapperProblemReplicate[601:149248] [Device UDID:	unknown]
2015-01-24 12:35:23.341 AutomapperProblemReplicate[601:149248] [Device Locale:	en_AU]
2015-01-24 12:35:23.347 AutomapperProblemReplicate[601:149248] [Device Date/Time:	24/01/2015 12:35:23 PM]
2015-01-24 12:35:23.349 AutomapperProblemReplicate[601:149248] [Bundle:	com.your-company.AutomapperProblemReplicate]
2015-01-24 12:35:23.660 AutomapperProblemReplicate[601:149248] critical: Stacktrace:
2015-01-24 12:35:23.661 AutomapperProblemReplicate[601:149248] critical:   at <unknown> <0xffffffff>
2015-01-24 12:35:23.661 AutomapperProblemReplicate[601:149248] critical:   at (wrapper managed-to-native) object.__icall_wrapper_mono_array_new_specific (intptr,int) <0xffffffff>
2015-01-24 12:35:23.662 AutomapperProblemReplicate[601:149248] critical:   at System.IO.StreamWriter.Initialize (System.Text.Encoding,int) [0x00017] in /Developer/MonoTouch/Source/mono/mcs/class/corlib/System.IO/StreamWriter.cs:77
2015-01-24 12:35:23.662 AutomapperProblemReplicate[601:149248] critical:   at System.IO.StreamWriter..ctor (string,bool,System.Text.Encoding,int) [0x00076] in /Developer/MonoTouch/Source/mono/mcs/class/corlib/System.IO/StreamWriter.cs:146
2015-01-24 12:35:23.662 AutomapperProblemReplicate[601:149248] critical:   at System.IO.StreamWriter..ctor (string,bool,System.Text.Encoding) [0x00000] in /Developer/MonoTouch/Source/mono/mcs/class/corlib/System.IO/StreamWriter.cs:123
2015-01-24 12:35:23.663 AutomapperProbl
emReplicate[601:149248] critical:   at System.IO.File.WriteAllText (string,string,System.Text.Encoding) [0x00000] in /Developer/MonoTouch/Source/mono/mcs/class/corlib/System.IO/File.cs:583
2015-01-24 12:35:23.663 AutomapperProblemReplicate[601:149248] critical:   at System.IO.File.WriteAllText (string,string) [0x00000] in /Developer/MonoTouch/Source/mono/mcs/class/corlib/System.IO/File.cs:578
2015-01-24 12:35:23.663 AutomapperProblemReplicate[601:149248] critical:   at AutomapperProblemReplicate.Tests.ReplicateTest.CauseCrash () [0x00033] in c:\Users\Jordan\Documents\Visual Studio 2013\Projects\AutomapperProblemReplicate\AutomapperProblemReplicate\Tests\ReplicateTest.cs:28
2015-01-24 12:35:23.663 AutomapperProblemReplicate[601:149248] critical:   at (wrapper runtime-invoke) object.runtime_invoke_dynamic (intptr,intptr,intptr,intptr) <0xffffffff>
2015-01-24 12:35:23.664 AutomapperProblemReplicate[601:149248] critical:   at <unknown> <0xffffffff>
2015-01-24 12:35:23.664 AutomapperProblemReplicate[601:149248] critical:   at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&) <0xffffffff>
2015-01-24 12:35:23.664 AutomapperProblemReplicate[601:149248] critical:   at System.Reflection.MonoMethod.Invoke (object,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo) [0x00044] in /Developer/MonoTouch/Source/mono/mcs/class/corlib/System.Reflection/MonoMethod.cs:230
2015-01-24 12:35:23.664 AutomapperProblemReplicate[601:149248] critical:   at System.Reflection.MethodBase.Invoke (object,object[]) [0x00000] in /Developer/MonoTouch/Source/mono/mcs/class/corlib/Syst
em.Reflection/MethodBase.cs:114
2015-01-24 12:35:23.665 AutomapperProblemReplicate[601:149248] critical:   at NUnit.Framework.Internal.Reflect.InvokeMethod (System.Reflection.MethodInfo,object,object[]) [0x00000] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/Reflect.cs:211
2015-01-24 12:35:23.665 AutomapperProblemReplicate[601:149248] critical:   at NUnit.Framework.Internal.Commands.TestMethodCommand.RunNonAsyncTestMethod (NUnit.Framework.Internal.TestExecutionContext) [0x00000] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/Commands/TestMethodCommand.cs:114
2015-01-24 12:35:23.665 AutomapperProblemReplicate[601:149248] critical:   at NUnit.Framework.Internal.Commands.TestMethodCommand.RunTestMethod (NUnit.Framework.Internal.TestExecutionContext) [0x0001d] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/Commands/TestMethodCommand.cs:90
2015-01-24 12:35:23.666 AutomapperProblemReplicate[601:149248] critical:   at
 NUnit.Framework.Internal.Commands.TestMethodCommand.Execute (NUnit.Framework.Internal.TestExecutionContext) [0x00000] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/Commands/TestMethodCommand.cs:66
2015-01-24 12:35:23.666 AutomapperProblemReplicate[601:149248] critical:   at NUnit.Framework.Internal.Commands.SetUpTearDownCommand.Execute (NUnit.Framework.Internal.TestExecutionContext) [0x00007] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/Commands/SetUpTearDownCommand.cs:84
2015-01-24 12:35:23.666 AutomapperProblemReplicate[601:149248] critical:   at NUnit.Framework.Internal.WorkItems.SimpleWorkItem.PerformWork () [0x00000] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/WorkItems/SimpleWorkItem.cs:64
2015-01-24 12:35:23.667 AutomapperProblemReplicate[601:149248] critical:   at NUnit.Framework.Internal.WorkItems.WorkItem.RunTest () [0x00059] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework
/Internal/WorkItems/WorkItem.cs:192
2015-01-24 12:35:23.667 AutomapperProblemReplicate[601:149248] critical:   at NUnit.Framework.Internal.WorkItems.WorkItem.Execute (NUnit.Framework.Internal.TestExecutionContext) [0x00070] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/WorkItems/WorkItem.cs:131
2015-01-24 12:35:23.667 AutomapperProblemReplicate[601:149248] critical:   at MonoTouch.NUnit.UI.TouchRunner.Run (NUnit.Framework.Internal.Test) [0x0002c] in /Developer/MonoTouch/Source/Touch.Unit/NUnitLite/TouchRunner/TouchRunner.cs:478
2015-01-24 12:35:23.668 AutomapperProblemReplicate[601:149248] critical:   at MonoTouch.NUnit.UI.TestCaseElement.Run () [0x00000] in /Developer/MonoTouch/Source/Touch.Unit/NUnitLite/TouchRunner/TestCaseElement.cs:80
2015-01-24 12:35:23.668 AutomapperProblemReplicate[601:149248] critical:   at MonoTouch.NUnit.UI.TestCaseElement/<TestCaseElement>c__AnonStorey0.<>m__0 () [0x00066] in /Developer/MonoTouch/Source/Touch.Unit/NUnitLite/TouchRunner/TestCaseEle
ment.cs:54
2015-01-24 12:35:23.668 AutomapperProblemReplicate[601:149248] critical:   at MonoTouch.Dialog.StringElement.Selected (MonoTouch.Dialog.DialogViewController,UIKit.UITableView,Foundation.NSIndexPath) [0x0000b] in /Developer/MonoTouch/Source/MonoTouch.Dialog/MonoTouch.Dialog/Elements.cs:713
2015-01-24 12:35:23.668 AutomapperProblemReplicate[601:149248] critical:   at MonoTouch.Dialog.DialogViewController.Selected (Foundation.NSIndexPath) [0x00029] in /Developer/MonoTouch/Source/MonoTouch.Dialog/MonoTouch.Dialog/DialogViewController.cs:548
2015-01-24 12:35:23.669 AutomapperProblemReplicate[601:149248] critical:   at MonoTouch.Dialog.DialogViewController/Source.RowSelected (UIKit.UITableView,Foundation.NSIndexPath) [0x00019] in /Developer/MonoTouch/Source/MonoTouch.Dialog/MonoTouch.Dialog/DialogViewController.cs:388
2015-01-24 12:35:23.669 AutomapperProblemReplicate[601:149248] critical:   at (wrapper runtime-invoke) object.runtime_invoke_dynamic (intptr,intptr,intptr,intptr) <0xffffffff>
2015-01-24 12
:35:23.669 AutomapperProblemReplicate[601:149248] critical:   at <unknown> <0xffffffff>
2015-01-24 12:35:23.670 AutomapperProblemReplicate[601:149248] critical:   at (wrapper managed-to-native) UIKit.UIApplication.UIApplicationMain (int,string[],intptr,intptr) <0xffffffff>
2015-01-24 12:35:23.671 AutomapperProblemReplicate[601:149248] critical:   at UIKit.UIApplication.Main (string[],intptr,intptr) [0x00005] in /Developer/MonoTouch/Source/monotouch/src/UIKit/UIApplication.cs:62
2015-01-24 12:35:23.671 AutomapperProblemReplicate[601:149248] critical:   at UIKit.UIApplication.Main (string[],string,string) [0x0001c] in /Developer/MonoTouch/Source/monotouch/src/UIKit/UIApplication.cs:45
2015-01-24 12:35:23.671 AutomapperProblemReplicate[601:149248] critical:   at AutomapperProblemReplicate.Application.Main (string[]) [0x00001] in c:\Users\Jordan\Documents\Visual Studio 2013\Projects\AutomapperProblemReplicate\AutomapperProblemReplicate\Main.cs:17
2015-01-24 12:35:23.672 AutomapperProblemReplicate[601:149248] critical:   at (wrapper runtime-invoke) object.runtime_invoke_dynamic (intptr,intptr,intptr,intptr) <0xffffffff>
```

**Comment by jbogard** [comment]
Well, I never compiled against Xamarin Unified. Added a new package that explicitly targets it, try this? http://www.nuget.org/packages/AutoMapper/4.0.0-ci1015

**Comment by jakkaj** [comment]
The problem still exists... although stack dump has changed a little. 

```
2015-01-27 16:15:58.662 AutomapperProblemReplicate[2059:652310] critical: Stacktrace:
2015-01-27 16:15:58.662 AutomapperProblemReplicate[2059:652310] critical:   at <unknown> <0xffffffff>
2015-01-27 16:15:58.663 AutomapperProblemReplicate[2059:652310] critical:   at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&) <0xffffffff>
2015-01-27 16:15:58.664 AutomapperProblemReplicate[2059:652310] critical:   at System.Reflection.MonoMethod.Invoke (object,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo) [0x00044] in /Developer/MonoTouch/Source/mono/mcs/class/corlib/System.Reflection/MonoMethod.cs:230
2015-01-27 16:15:58.664 AutomapperProblemReplicate[2059:652310] critical:   at System.Reflection.MethodBase.Invoke (object,object[]) [0x00000] in /Developer/MonoTouch/Source/mono/mcs/class/corlib/System.Reflection/MethodBase.cs:114
2015-01-27 16:15:58.665 AutomapperProblemReplicate[2059:652310] critical:   at NUnit.Framework.Internal.Reflect.InvokeMethod (System.Reflect
ion.MethodInfo,object,object[]) [0x00000] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/Reflect.cs:211
2015-01-27 16:15:58.665 AutomapperProblemReplicate[2059:652310] critical:   at NUnit.Framework.Internal.Commands.TestMethodCommand.RunNonAsyncTestMethod (NUnit.Framework.Internal.TestExecutionContext) [0x00000] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/Commands/TestMethodCommand.cs:114
2015-01-27 16:15:58.666 AutomapperProblemReplicate[2059:652310] critical:   at NUnit.Framework.Internal.Commands.TestMethodCommand.RunTestMethod (NUnit.Framework.Internal.TestExecutionContext) [0x0001d] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/Commands/TestMethodCommand.cs:90
2015-01-27 16:15:58.666 AutomapperProblemReplicate[2059:652310] critical:   at NUnit.Framework.Internal.Commands.TestMethodCommand.Execute (NUnit.Framework.Internal.TestExecutionContext) [0x00000] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/Commands/TestMethodCommand.cs:66
2015-01-27 16:15:58.667 AutomapperProblemReplicate[2059:652310] critical:   at NUnit.Framework.Internal.Commands.SetUpTearDownCommand.Execute (NUnit.Framework.Internal.TestExecutionContext) [0x00007] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/Commands/SetUpTearDownCommand.cs:84
2015-01-27 16:15:58.668 AutomapperProblemReplicate[2059:652310] critical:   at NUnit.Framework.Internal.WorkItems.SimpleWorkItem.PerformWork () [0x00000] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/WorkItems/SimpleWorkItem.cs:64
2015-01-27 16:15:58.668 AutomapperProblemReplicate[2059:652310] critical:   at NUnit.Framework.Internal.WorkItems.WorkItem.RunTest () [0x00059] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/WorkItems/WorkItem.cs:192
2015-01-27 16:15:58.669 AutomapperProblemReplicate[2059:652310] critical:   at NUnit.Framework.Internal.WorkItems.WorkItem.Execute (NUnit.Framework.Internal.TestExecutionContext) [0x00070] in /Developer/MonoTouch/Source/NUnitLite/NUnitLite-1.0.0/src/framework/Internal/WorkItems/WorkItem.cs:131
2015-01-27 16:15:58.669 AutomapperProblemReplicate[2059:652310] critical:   at MonoTouch.NUnit.UI.TouchRunner.Run (NUnit.Framework.Internal.Test) [0x0002c] in /Developer/MonoTouch/Source/Touch.Unit/NUnitLite/TouchRunner/TouchRunner.cs:478
2015-01-27 16:15:58.670 AutomapperProblemReplicate[2059:652310] critical:   at MonoTouch.NUnit.UI.TestCaseElement.Run () [0x00000] in /Developer/MonoTouch/Source/Touch.Unit/NUnitLite/TouchRunner/TestCaseElement.cs:80
2015-01-27 16:15:58.670
 AutomapperProblemReplicate[2059:652310] critical:   at MonoTouch.NUnit.UI.TestCaseElement/<TestCaseElement>c__AnonStorey0.<>m__0 () [0x00066] in /Developer/MonoTouch/Source/Touch.Unit/NUnitLite/TouchRunner/TestCaseElement.cs:54
2015-01-27 16:15:58.671 AutomapperProblemReplicate[2059:652310] critical:   at MonoTouch.Dialog.StringElement.Selected (MonoTouch.Dialog.DialogViewController,UIKit.UITableView,Foundation.NSIndexPath) [0x0000b] in /Developer/MonoTouch/Source/MonoTouch.Dialog/MonoTouch.Dialog/Elements.cs:713
2015-01-27 16:15:58.671 AutomapperProblemReplicate[2059:652310] critical:   at MonoTouch.Dialog.DialogViewController.Selected (Foundation.NSIndexPath) [0x00029] in /Developer/MonoTouch/Source/MonoTouch.Dialog/MonoTouch.Dialog/DialogViewController.cs:548
2015-01-27 16:15:58.672 AutomapperProblemReplicate[2059:652310] critical:   at MonoTouch.Dialog.DialogViewController/Source.RowSelected (UIKit.UITableView,Foundation.NSIndexPath) [0x00019] in /Developer/MonoTouch/Source/MonoTouch.Dialog/MonoTouch.Dialog/DialogViewController.cs:388
2015-01-27 16:15:58.673 AutomapperProblemReplicate[2059:652310] critical:   at (wrapper runtime-invoke) object.runtime_invoke_dynamic (intptr,intptr,intptr,intptr) <0xffffffff>
2015-01-27 16:15:58.673 AutomapperProblemReplicate[2059:652310] critical:   at <unknown> <0xffffffff>
2015-01-27 16:15:58.674 AutomapperProblemReplicate[2059:652310] critical:   at (wrapper managed-to-native) UIKit.UIApplication.UIApplicationMain (int,string[],intptr,intptr) <0xffffffff>
2015-01-27 16:15:58.675 AutomapperProblemReplicate[2059:652310] critical:   at UIKit.UIApplication.Main (string[],intptr,intptr) [0x00005] in /Developer/MonoTouch/Source/monotouch/src/UIKit/UIApplication.cs:62
2015-01-27 16:15:58.675 AutomapperProblemReplicate[2059:652310] critical:   at UIKit.UIApplication.Main (string[],string,string) [0x0001c] in /Developer/MonoTouch/Source/monotouch/src/UIKit/UIApplication.cs:45
2015-01-27 16:15:58.676 AutomapperProblemReplicate[2059:652310] critical:   at AutomapperProblemReplicate.Application.Main (string[]) [0x00001] in c:\Users\Jordan\Documents\Visual Studio 2013\Projects\AutomapperProblemReplicate\AutomapperProblemReplicate\Main.cs:17
2015-01-27 16:15:58.676 AutomapperProblemReplicate[2059:652310] critical:   at (wrapper runtime-invoke) object.runtime_invoke_dynamic (intptr,intptr,in
tptr,intptr) <0xffffffff>
2015-01-27 16:15:58.677 AutomapperProblemReplicate[2059:652310] critical: 
```

**Comment by jakkaj** [comment]
I tried firing up the source to see if I can dig a little deeper, but it's not building on my machine. I'm on the clock so don't have much time spend getting it running sorry.

**Comment by jbogard** [comment]
No worries, I can give it a shot on my box.

**Comment by jbogard** [comment]
Well nuts, works just fine on my simulator too, but I don't have a 64-bit device :(

**Comment by jbogard** [comment]
And the 64-bit simulator it seems.

**Comment by swerdk** [comment]
@jbogard, firstly thank you so much for this awesome package. I have been using it joyfully :-)

I also got issues with AutoMapper on 64bit devices. I tried http://www.nuget.org/packages/AutoMapper/4.0.0-ci1015, but still not working.

Any ideas?

**Comment by swerdk** [comment]
@jbogard  - Update

I just changed my Linker to "Link all assemblies" for platform iPhone and Debug and I was able to get this exception when app crashed:

"This type is not supported on this platform IPlatformSpecificMapperRegistry"

Before changing the Linker option, my app just crashed without any warning or exception thrown.

**Comment by jbogard** [comment]
OK so I got something working on my 64-bit device. Can you try 1019?

**Comment by swerdk** [comment]
Just tried with 1019 and still got the same exception: This type is not supported on this platform IPlatformSpecificMapperRegistry

**Comment by jbogard** [comment]
Can you provide repro steps? I just did "Create new Xamarin.Forms project, add AutoMapper package to both the core and iOS assemblies, and do CreateMap/Map in the core project"

**Comment by swerdk** [comment]
I Just created a new Xamarin.Forms project, added the AutoMapper and did CreateMap/Map and it work, just as you said.

I tried the same for my current project. Cleaned project, added the new 1019 package to both core and iOS, build and it worked. That's amazing!

Seems that the issue has been solved :-)

Thank you Jimmy for your time and the great work on keeping AutoMapper the coolest package on NuGet.

May you have a nice and awesome day!

**Comment by jbogard** [comment]
Awesome! It seems like with Xamarin, the solution is often "turn it off and on again"

**Comment by jakkaj** [comment]
Thanks Jimmy - much appreciated. Will give it a shot later. 

Sent from Outlook




On Thu, Jan 29, 2015 at 2:30 PM -0800, "Jimmy Bogard" <notifications@github.com> wrote:












Closed #669.

**Comment by wallism** [comment]
Had the same problem, installed 4.0.0-ci1031, didn't work still :-( 
...uninstalled the app and restarted everything...
Works!
THANKS!! :-)
